.github/scripts/run-yaml-api-tests.py

```diff
@@ -6,8 +6,8 @@
 
 import sys
 
-def run_build_recipes_ini_api_tests():
-    """ Test config_{build, build_recipes}.ini APIs """
+def run_build_recipes_yaml_api_tests():
```

Comment:
These tests are extremely basic. Frankly, I have little experience writing pytests so I figured I would quickly get something in and it could be later cleaned up (in a more portable way, and in a way that will result in less pain moving forward). Im happy to move some of these files out of this directory, but I think for this PR we can keep these where they are and make a future TODO.

Reply:
Sounds good.  Like I said, I don't mind helping move and/or rewrite them to be easier to maintain.